lms/djangoapps/instructor/views/api.py

```diff
@@ -2654,8 +2710,31 @@ def spoc_gradebook(request, course_id):
         }
         for student in enrolled_students
     ]
+    return student_info, page_num, total_pages, next_offset, previous_offset
+
+
+# ---- Gradebook (shown to small courses only) ----
+@cache_control(no_cache=True, no_store=True, must_revalidate=True)
+@require_level('staff')
+def spoc_gradebook(request, course_id):
+    """
+    Show the gradebook for this course:
+    - Only shown for courses with enrollment < settings.FEATURES.get("MAX_ENROLLMENT_INSTR_BUTTONS")
+    - Only displayed to course staff
+    """
+    course_key = SlashSeparatedCourseKey.from_deprecated_string(course_id)
```

Comment:
@giocalitri this line was from previous code not in my implantation. But if you mentioned here let me fix it.